src/App.jsx

```diff
@@ -18,6 +19,9 @@ function App() {
         <Route path="applicant" element={<div>APPLICANT</div>} />
         <Route path="interview" element={<div>INTERVIEW</div>} />
         <Route path="application" element={<div>APPLICATION</div>} />
+
+        {/* apply pages */}
+        <Route path="/apply/test" element={<Apply2/>} />
```

Comment:
```suggestion
        <Route path="apply/test" element={<Apply2/>} />
```
‼️`/apply/test`는 절대경로라 라우터 트리 구조가 깨지기 때문에, 상대경로인 `apply/test`로 수정 부탁드립니다!